### PR TITLE
ENH added xpa

### PR DESCRIPTION
**Pull request by beckermr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by beckermr** [comment]
I reported the patch here: https://github.com/ericmandel/xpa/issues/10

**Comment by beckermr** [comment]
Alright @conda-forge/staged-recipes! This one is ready for review!

File: recipes/xpa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - add-ldflags-shlib.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/xpa/meta.yaml]:
```suggestion
    - {{ compiler('cxx') }}
    - make
```

File: recipes/xpa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set name = "xpa" %}
+{% set version = "2.1.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ericmandel/xpa/archive/v2.1.19.tar.gz
```
Comment (chrisburr) [inline recipes/xpa/meta.yaml]:
```suggestion
  url: https://github.com/ericmandel/xpa/archive/v{{ version }}.tar.gz
```

**Comment by beckermr** [comment]
OK @chrisburr @conda-forge/staged-recipes. This is ready for another look!

**Comment by beckermr** [comment]
Thank you!

**Pull request merged**